src/api/initialDataTypes.js

```diff
@@ -78,6 +78,13 @@ export type InitialDataRealm = {|
   realm_uri: string,
   realm_video_chat_provider: string,
   realm_waiting_period_threshold: number,
+  /**
+   * Added in server version 2.2, feature level 1. Older servers
+   * don't send this.
+   * See https://github.com/zulip/zulip/commit/e3b90a5ec8 for the
+   * server-side implementation.
```

Comment:
This one isn't strictly documented (the initial data is pretty undocumented in general); but this field is exactly like the corresponding one in `/server_settings`. So I think the cleanest thing is to say that -- like
> Same meaning as in the `server_settings` response:
>   https://zulipchat.com/api/server-settings